### PR TITLE
Implemented reading guide quality values from real systems.

### PR DESCRIPTION
**Pull request by jluhrs**
Values are now read from EPICS channels. I also added the possibility to have different behaviors for North and South TCS (although for now they share the same implementation)

### CODE REVIEW

File: modules/server/src/main/scala/navigate/server/Systems.scala [hunk 1]
```diff
@@ -41,12 +41,27 @@ object Systems {
...
-          p2  <- WfsEpicsSystem.build(epicsSrv, "PWFS2", readTop(tops, "pwfs2".refined))
-          oi  <- WfsEpicsSystem.build(epicsSrv,
-                                      "OIWFS",
-                                      readTop(tops, "oiwfs".refined),
-                                      "dc:initSigInit.MARK".refined
+          p1  <- WfsEpicsSystem.build(
+                   epicsSrv,
+                   "PWFS1",
+                   readTop(tops, "pwfs1".refined),
+                   "dc:fgDiag6P1.VALH".refined,
```
Comment (cquiroz) [inline modules/server/src/main/scala/navigate/server/Systems.scala:48]: No `top` for these?

File: modules/server/src/main/scala/navigate/server/tcs/WfsEpicsSystem.scala [hunk 2]
```diff
@@ -47,15 +48,26 @@ object WfsEpicsSystem {
         new WfsEpicsImpl[F](channels),
         timeout
       )
+
+    override def getQualityStatus: WfsQualityStatus[F] = new WfsQualityStatus[F] {
+      override val flux: VerifiedEpics[F, F, Int]                 =
+        VerifiedEpics.readChannel(channels.telltale, channels.flux)
+      override val centroidDetected: VerifiedEpics[F, F, Boolean] = VerifiedEpics
```
Comment (cquiroz) [inline modules/server/src/main/scala/navigate/server/tcs/WfsEpicsSystem.scala:55]: Are centroids calculated directly on the TCS?

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/server/src/main/scala/navigate/server/Systems.scala (hunk 1 above)
Comment (jluhrs) [reply modules/server/src/main/scala/navigate/server/Systems.scala:48]: They are relative to the `top` given in the previous parameter.

File: modules/server/src/main/scala/navigate/server/tcs/WfsEpicsSystem.scala (hunk 2 above)
Comment (jluhrs) [reply modules/server/src/main/scala/navigate/server/tcs/WfsEpicsSystem.scala:55]: No, both values come from the WFS. Which is why they are in `WfsEpicsSystem` and not `TcsEpicsSystem`.

**Pull request merged**